sufia-models/lib/generators/sufia/models/install_generator.rb

```diff
@@ -127,4 +128,9 @@ def upload_to_collection_config
   def minimagic_config
     generate 'sufia:models:minimagick_config'
   end
+
+  # Adds geonames_username initialization
+  def upload_to_collection_config
```

Comment:
The method on line 123 is already named this. Rename to something like `geonames_config`.